module/MODULE.md

```diff
@@ -0,0 +1,68 @@
+# Feature Store Module
+
+## Dependencies
+
+```shell
+sudo apt install flex bison pahole libssl-dev libelf-dev
+```
+
+## Build and Deploy Kernel
+
+This should mostly be handled by `make -C module -j$(nproc) ${MODULE}/lib`.
+However first you need to introduce environment variables that are going to aid this.
+
+```shell
+export SUDO="sudo"
+export MODULE=/
+export KHEADS=/usr/src/linux-headers-6.12.0-gadc218676eef
```

Comment:
the 6.12 hash is `adc218676eef25575469234709c2d87185ca223a`, is the `g` intentional?

Reply:
Pretty sure it stands for generic. Run uname-r on caferri or moriarty.

Reply:
that tracks